managed/src/main/java/com/yugabyte/yw/models/HealthCheck.java

```diff
@@ -23,6 +27,23 @@
 public class HealthCheck extends Model {
   public static final Logger LOG = LoggerFactory.getLogger(HealthCheck.class);
 
+  public static class details {
+    public static class nodeData {
+      public String node;
+      public String process;
+      public String timestamp;
+      public String node_name;
+      public Boolean has_error;
+      public List<String> details;
+      public String message;
+    }
+
+    public Date timestamp;
+    public List<nodeData> data = new ArrayList<>();
+    public String yb_version;
+    public Boolean error;
```

Comment:
@sb-yb 
```{"error": false, "data": [ {"node":"n", "has_error": true, "message":"c" } ] }
2021-06-22 00:23:03.788 [info] HikariCPModule.scala:79 [application-akka.actor.default-dispatcher-5] Shutting down connection pool.
[error] Test com.yugabyte.yw.commissioner.HealthCheckerTest.testSingleUniverseNoEmail failed: java.lang.IllegalArgumentException: Unrecognized field "error" (class com.yugabyte.yw.models.HealthCheck$details), not marked as ignorable (4 known properties: "data", "has_error", "timestamp", "yb_version"])
[error]  at [Source: UNKNOWN; line: -1, column: -1] (through reference chain: com.yugabyte.yw.models.HealthCheck$details["error"]), took 0.767```
it should have been has_error ( checked with jitendra's DB entry ) I will setup healthcheck and validate but test case is only passing `error`  , not sure how it was working before if the field which is passed is error.